### PR TITLE
Conversation context menu

### LINKED ISSUE #727: Conversation context menu
Replicate the context menu long press interaction on conversations as implemented in iMessage, Messenger, WhatsApp, and Telegram.

- [x] Long press a conversation in the conversation list to reveal a context menu (replaces long-press to pin)
- [x] Add "Pin" to context menu
- [x] Add "Mark as Unread" to context menu
- [x] Add "Delete" to context menu
- [x] Display auxiliary preview of the conversation

### PR DESCRIPTION
**Pull request by lourou**
Conversation context menu implementation with auxiliary preview

Closes #727

### CODE REVIEW

File: components/ConversationContextMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+  useSharedValue,
+  withTiming,
+} from "react-native-reanimated";
+
+import { useIsSplitScreen } from "../screens/Navigation/navHelpers";
+
+const AnimatedBlurView =
+  Platform.OS === "ios"
+    ? Animated.createAnimatedComponent(BlurView)
+    : Animated.createAnimatedComponent(View);
```
Comment (thierryskoda) [inline components/ConversationContextMenu.tsx:41]: Should we make a BlurView component in `components` that we can reuse. So if later BlurView is fixed on Android we can just change it in one file

**Review by thierryskoda — APPROVED** [review]
Small comment but the rest looks good to me!

**Comment by thierryskoda** [comment]
Missing some context maybe but also what we could do is in Chat create a components just for the messages so we can display that instead of the whole Chat component. Might help with performance.

File: components/ConversationContextMenu.tsx (hunk 1 above)
Comment (lourou) [reply components/ConversationContextMenu.tsx]: Yes, good one, just pushed this change

**Comment by lourou** [comment]
> Missing some context maybe but also what we could do is in Chat create a components just for the messages so we can display that instead of the whole Chat component. Might help with performance.

I've started working on a Chat component refactor on a separate branch/PR, so we can improve perf on the auxiliary preview.

**Pull request merged**